custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -306,7 +321,11 @@ def get_drtb_hiv_referral_case_structure(self, episode_structure):
                 related_type=EPISODE_CASE_TYPE,
             )],
         }
-
+        if self.existing_drtb_hiv_case:
```

Comment:
@nickpell we need to do this for any case being created by migration. was a bit surprised you hadn't already accounted for this given the similarities to other case types